SmurfEngine/Characters/Character.cs

```diff
@@ -41,12 +40,12 @@ public Character(string name, int health, Inventory inventory)
             this.Health = health;
             this.Inventory = inventory;
 
-            this.stats[Stats.STR] = new Stat("STR", 0, 0);
-            this.stats[Stats.INT] = new Stat("INT", 0, 0);
-            this.stats[Stats.WIS] = new Stat("WIS", 0, 0);
-            this.stats[Stats.DEX] = new Stat("DEX", 0, 0);
-            this.stats[Stats.CON] = new Stat("CON", 0, 0);
-            this.stats[Stats.CHA] = new Stat("CHA", 0, 0);
+            this.stats.Add(StatType.STR.ToString(), new Stat(StatType.STR.ToString(), 0, 0));
```

Comment:
This is being used in both constructors.  It would be better to have this as a separate method to load the stats on initialization.

```csharp        
private void InitializeStats()
{
    foreach (var statName in Enum.GetNames(typeof(StatType)))
        this.stats.Add(statName, new Stat(statName, 0, 0));
}
```